esphome/components/mitsubishi_itp/muart_rawpacket.h

```diff
@@ -35,15 +35,19 @@ enum class GetCommand : uint8_t {
   ERROR_INFO = 0x04,
   STATUS = 0x06,
   RUN_STATE = 0x09,
-  A_9 = 0xa9
+  KUMO_GET_ADAPTER_STATE = 0xa9,
+  KUMO_AB = 0xab,
```

Comment:
I mentioned this elsewhere, but let's name this something without "KUMO" if we can think of something, so we're not confusing anyone that there's an actual association with the Kumo service.

Reply:
Taking suggestions 😅.

I'm not aware if the MHK2 even exists outside of the North American market. A quick Google search didn't give me any results, and I'm unsure if MELCloud units even _have_ a second CN105 port on them. These packets may really be Kumo exclusive only.

Point taken about the name, though I will note that this really is an internal designation. Other than the flag (discussed in a prior message), I don't think we'd expose the "kumo" name anywhere except our documentation on the IT protocol.

Reply:
Taking another look now, let's just drop the "KUMO".  `THERMOSTAT_SENSOR_STATUS`, `THERMOSTAT_HELLO`, and `THERMOSTAT_STATE_SYNC` all seem pretty reasonable.  These packets are *specifically* for communication between the thermostat and the ESP (rather than communication between a controller and heat pump).

Just to keep things consistent let's name `0xa9` as `THERMOSTAT_STATE_SYNC` as well.